### PR TITLE
Rename everything 'resin' to 'balena'

### PR DESCRIPTION
**Pull request by thgreasi**
Resolves: #20
Change-type: major
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: node_js
 before_install:
-  - npm -g install npm@latest-4
+  - npm -g install npm@6
 node_js:
+  - "10"
```
Comment (sradevski) [inline .travis.yml:6]: Since node v6 has been deprecated, maybe we have 8, 10, 12 instead?

**Review by sradevski — APPROVED** [review]
LGTM, you can just adjust the travis nodejs versions if you wish.

File: .travis.yml (hunk 1 above)
Comment (thgreasi) [reply .travis.yml:6]: Will do :+1:

**Comment by LucianBuzzo** [comment]
@thgreasi Is this PR ready to go now? If so I recommend merging this and doing the TS conversion in a separate PR.

**Pull request merged**